system/system.proto

```diff
@@ -118,6 +118,9 @@ message RebootResponse {
 // A RebootMethod determines what should be done with a target when a Reboot is
 // requested.  Only the COLD method is required to be supported by all
 // targets.  Methods the target does not support should result in failure.
+// 
+// If a target does not support a method, `INVALID_ARGUEMENT` should be
```

Comment:
Should i also say that INVALID_ARGUMENT should be returned if the default/unknown method is provided or is that implicit?

Reply:
Added explicity reference for both scenarios.